cluster/manifests/psp/pod_security_policy.yaml

```diff
@@ -7,6 +7,9 @@ spec:
     rule: RunAsAny
   privileged: true
   allowPrivilegeEscalation: true
+  allowedCapabilities:
+  - IPC_LOCK
+  - SYS_RESOURCE
```

Comment:
this could be harmful, because the process is allowed to increase RLIMIT_NPROC and RLIMIT_NOFILE

What is the reason you suggest this?